package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "3.8.22",
+  "version": "3.9.0",
```

Comment:
This is a somewhat-significant change, and patch version 22 was getting pretty high, so I updated the minor version.